src/index.js

```diff
@@ -32,7 +32,7 @@ import AdminLayout from './layouts/Admin.jsx';
 import { getConfig } from './utils/getConfig';
 
 import './index.css';
-import 'antd/dist/antd.css';
+// import 'antd/dist/antd.css';
```

Comment:
Context? Wouldn't removing be better?

Reply:
Had to remove these lines due to dependent library change. Removed the lines now. Thanks